RELEASING.md

```diff
@@ -17,10 +17,23 @@ Go the project root directory and run
 make release
 ```
 
-> **_NOTE:_** If you are not a maintainer, you might want to set the `REMOTE` env variable to your preferred remote name. The `create_pull_requests` target might fail and you will need to create the PRs manually from the branches that were pushed.
+> **_NOTE:_** If you are not a maintainer, you might want to set the `REMOTE` env variable to your preferred remote name. The `create_pull_requests` target might fail, then you will need to create the PRs manually from the branches that were pushed.
 
 ### For Maintainers
 
-Once a release is prepared 2 Pull Request will be generated. They have to be merged in order. First the one for the release and then the one for the next development version (it should be a draft PR). The automation will take care of the rest.
+Once a release is prepared 2 pull request will be generated. They have to be merged in order. First the one for the release and then the one for the next development version (it should be a draft PR). The automation will take care of the rest.
 
 > **_NOTE:_** Please review the changes in case there are errors if the PR were modified after the automation ran.
+
+#### Versioning
+
+This project follows (semver)[https://semver.org/] versioning standards.
```

Comment:
"versioning standards" is redundant :) Semver = versioning standard.

Reply:
I've made it explicit now.